client/src/mdn.scss

```diff
@@ -77,4 +77,614 @@ div.content {
   .live-sample-frame {
     max-width: 42rem;
   }
+
```

Comment:
Can you cut all the additions inside `mdn.scss` and put them into a file called `bcd.scss` and then import it from the top of `document.js` like so:

```
import('./bcd.scss');
```

Eventually, we might lazy-load that CSS too so it's only loaded lazily on documents that have BCD tables. But this'll tie us over. Also, when time allows @schalkneethling will take over that file and do whatever magic he needs to do with it.